### PR TITLE
Update core to 0.95.0

### LINKED ISSUE #1633: New encryption implementation
Connects to #1547 
Connects to #1398

### PR DESCRIPTION
**Pull request by beeender**
Just to be prepared for new core release.

Fixes #1633

### CODE REVIEW

File: realm/realm-jni/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import java.security.MessageDigest
 
-ext.coreVersion = '0.94.4'
+ext.coreVersion = '0.95.0'
 // empty or comment out this to disable hash checking
 ext.coreSha256Hash = 'feeae46e5ebdb7ce912b3f111b6a044c4dd0ef7b7eac9b313f8c7a40664d9fae'
```
Comment (zaki50) [inline realm/realm-jni/build.gradle]: `coreSha256Hash` should be updated.

Comment (beeender) [inline realm/realm-jni/build.gradle]: Well, it is not released yet. I can make up one but it won't help too much :grin:

Comment (zaki50) [inline realm/realm-jni/build.gradle]: Oh, I see

File: changelog.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@
  * Fixed some memory leaks when an Exception is thrown (#1730).
  * Fixed a memory leak when using relationships. (#1285)
  * RealmQuery.isNull() and RealmQuery.isNotNull() now throw IllegalArgumentException instead of RealmError if the fieldname is a linked field and the last element is a link (#1693).
+ * Removed RealmEncryptionNotSupportedException since encryption implementation changed in Realm underlying engine. Encryption is supported on any devices now.
```
Comment (bdash) [inline changelog.txt]: s/any/all/

Grammar elsewhere in this point could be improved too.

**Comment by cmelchior** [comment]
Ready for review @realm/java

File: changelog.txt [hunk 3]
```diff
@@ -1,11 +1,14 @@
 0.85.0
+ * BREAKING CHANGE: Removed RealmEncryptionNotSupportedException since the encryption implementation changed in Realms underlying storage engine. Encryption is now supported now on all devices.
```
Comment (zaki50) [inline changelog.txt]: now supported now -> now supported

**Comment by zaki50** [comment]
:+1:

File: changelog.txt [hunk 4]
```diff
@@ -1,11 +1,14 @@
 0.85.0
+ * BREAKING CHANGE: Removed RealmEncryptionNotSupportedException since the encryption implementation changed in Realms underlying storage engine. Encryption is now supported on all devices.
```
Comment (kneth) [inline changelog.txt]: "Realms" -> "Realm's"

**Comment by kneth** [comment]
:+1:

**Comment by dalinaum** [comment]
:+1:

**Pull request merged**